ksqldb-rest-client/src/main/java/io/confluent/ksql/rest/client/KsqlTargetUtil.java

```diff
@@ -69,6 +70,41 @@ public static List<StreamedRow> toRows(final Buffer buff) {
     return rows;
   }
 
+  public static List<StreamedRow> toRowsFromProto(final Buffer buff) {
+    final List<StreamedRow> rows = new ArrayList<>();
+    boolean headerRead = false;
+    int begin = 0;
+
+    for (int i = 0; i <= buff.length(); i++) {
```

Comment:
This method deserves a detailed javadoc.

Reply:
done